server/sonar-server/src/main/java/org/sonar/ce/queue/CeQueueImpl.java

```diff
@@ -213,10 +219,15 @@ public CeTask apply(@Nonnull CeQueueDto dto) {
         builder.setComponentUuid(componentUuid);
         ComponentDto component = componentDtoByUuid.get(componentUuid);
         if (component != null) {
+          builder.setOrganizationUuid(component.getOrganizationUuid());
           builder.setComponentKey(component.getKey());
           builder.setComponentName(component.name());
         }
       }
+      // fixme this should be set from the CeQueueDto
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)